src/rpc/handlers/AccountObjects.cpp

```diff
@@ -139,10 +143,9 @@ doAccountNFTs(Context const& context)
                 nextKey.key});
         }
         else
-            break;
+            cp.reset();
```

Comment:
Why change this to a `reset()` over a `break`?

Reply:
Because I changed `while(true)` to `while(cp)`